tests/pos/i480a.scala

```diff
@@ -0,0 +1,62 @@
+package dotty.tools
+package dotc
+package core
+
+import Types._, Contexts._, Symbols._, Denotations._, SymDenotations._, StdNames._, Names._
+import Flags._, Scopes._, Decorators._, NameOps._, util.Positions._
+import pickling.UnPickler.ensureConstructor
+import scala.annotation.{ switch, meta }
+import scala.collection.{ mutable, immutable }
+import PartialFunction._
+import collection.mutable
+import scala.reflect.api.{ Universe => ApiUniverse }
+
+object Definitions {
```

Comment:
I propose to either minimize this test, and make it not depend on Dotty internals
 or replace it with https://github.com/dotty-staging/dotty/commit/287fd1a2044838654146568561d74543a2f0b761.